lib/coinbase/address/wallet_address.rb

```diff
@@ -95,6 +95,18 @@ def trade(amount, from_asset_id, to_asset_id)
       trade
     end
 
+    # Signs the given unsigned payload.
+    # @param unsigned_payload [String] The hex-encoded hashed unsigned payload for the Address to sign.
+    # @return [Coinbase::PayloadSignature] The payload signature
+    def sign(unsigned_payload:)
```

Comment:
Would we want this to be `sign_payload` to mirror `payload_signatures`?

Also, I don't know if I would expect `sign` to create a resource and be async by default, `sign_payload` might make that a bit more obvious (since all other `sign` methods are in-memory).

Reply:
Updated to `sign_payload`

Reply:
And the truth is that I'm worried because I also lost, I went bankrupt, because I invested in Block Five and right now they're giving me a refund and I can't access it, I don't know. But today I'm the best, excuse me, but if you can help me, please, I really admire you all a lot and today I'm just asking for your help, I hope. I can help you very humbly, I ask you.